Sources/Sentry/include/SentryEnvelope.h

```diff
@@ -19,6 +22,12 @@ SENTRY_NO_INIT
  */
 @property (nonatomic, readonly, copy) NSString *_Nullable eventId;
 
+/**
+ * Describes the system SDK. Although this can be nil it is recommended to specifiy it when sending
+ * an envelope.
```

Comment:
M: this comment is odd to me, I'd describe in a different way, it is required to set if you are creating an envelope.
if you are parsing an envelope that is already cached, you should not overwrite it even if it's null.

I'd understand from the current text that if it's null and I'm sending it, I need to specify it

Reply:
I removed it and added a comment to `SentrySdkInterface`